group_vars/all.yml

```diff
@@ -1507,9 +1507,10 @@ nginx_blacklist_vhost_check_include_file: "{{ nginx_etc_dir }}/magento_url_black
 # --------  Elasticsearch  --------
 # ---------------------------------
 
-# Install elasticsearch 6.x as this is the latest version fully supported by
-# ElasticSuite. There's already 7.x though...
-repo_elasticsearch_variants_enabled: ['6']
+# Install elasticsearch 7.x as this is the latest version fully supported by
+# ElasticSuite. There's already 8.x though...
+repo_elasticsearch_variants_enabled: ['7']
+# mageops_elasticsearch_version: 8.4
```

Comment:
Is this used anywhere?

Reply:
Yes, depends on this you assign different repo of ES